operator-framework/pom.xml

```diff
@@ -17,7 +17,11 @@
       <groupId>io.javaoperatorsdk</groupId>
       <artifactId>operator-framework-core</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-httpclient-jdk</artifactId>
```

Comment:
Should not be this the vertx client?

Reply:
As mentioned, I would like to have tests with different clients to see if there is any impact